data/db_projectHelpers.js

```diff
@@ -0,0 +1,26 @@
+const knex = require('knex');
+
+const db_config = require('../knexfile.js');
+
+const db = knex(db_config.development);
+
+module.exports = {
+
+    getProjects: () => {
+        return db('project');
+    },
+
+    // getProjectById: (id) => {
+    //     return db('project')
+    //         .where('id', id)
+    //         .first()
+    //         .then(project => {
+    //             db('action')
+    //                 .where('project_id', id)
+    //         })
+    // },
```

Comment:
With your current get project by id logic in the server endpoint, it could look like this:
```javascript
getProjectById: (id) => {
  db('project')
    .first()
    .then(project => {
      db('action')
        .where('project_id', id)
        .then(action => {
          return { ...project, action };
        })
    })
},
```

Reply:
Note that this is still not functioning quite right due to what I pointed out in the other file!